### ISSUE TITLE
How to change the color of the model from blue to red?

### ISSUE THREAD

**Issue by tobybonhof** [dbrant/ModelViewer3D#2]

**Comment by dbrant** [comment]
The `Light.java` class specifies an `ambientColor` which contains RGB values. You can play around with these values to change the ambient color.

**Comment by tobybonhof** [comment]
Yes, i have done that. But it does not actually change the model color right? because from some angles the model still shows a blueish color

**Comment by dbrant** [comment]
The model itself has no color -- it's just a mesh shape. The color of the ambient light determines the color in which the model is rendered.  If you set it properly, you should not see any blue remnants.  For example, something like `{0.5f, 0.0f, 0.0f}` should give you a red color.

**Comment by tobybonhof** [comment]
This doesn't fully work unfortunately there are still some blue glares from certain angles. Does the environment color (floor color) have any impact on this? If so is it possible to turn off the realistic glow / glare effect all together? :)

**Comment by dbrant** [comment]
I'm afraid I have no idea where that blue (or yellow?) color might be coming from. Can you share the model that you're using in the screenshot, so that I can attempt to reproduce your exact conditions?

**Comment by dbrant** [comment]
Are you quite sure that you didn't make any other modifications to the code? (including the shader .glsl files?)  When I try using the parameters you pasted, I'm not seeing any blue colors, and I don't see how you could be getting them either, unless something else has been modified.

Perhaps try cleaning/rebuilding the project, or even resetting the repository to its initial state, and start fresh.

On a side note, it looks like you're setting the specular color to gray, while keeping the diffuse color as white. This will produce a rather unnatural appearance, unless that's what you're going for.

**Comment by tobybonhof** [comment]
So I have tested this code again on my Google pixel and it works (Android 8) I think it has something to do with Android 6 as the screenshot is from a Nexus 5 (Android 6). 

And the white and gray colors, you are right these need to be adjusted.
